src/games/strategy/engine/history/History.java

```diff
@@ -28,6 +28,8 @@
 import games.strategy.triplea.ui.history.HistoryPanel;
 
 public class History extends DefaultTreeModel {
+  private static final long serialVersionUID = -1769876896869L;
```

Comment:
Please try to split up independent changes into different PRs. Any objections is going to hold up all work in this PR. In addition it is difficult to review when the diff is really long, and when there are many different changes.